### DISCUSSION TITLE
navigateTo does not work for same path.

### DISCUSSION THREAD

**Discussion by upgreidas** [nuxt/nuxt#16742] [Questions]
Let's say  current route is /categories/all and I want to navigate to /categories/all?region=US. I use navigateTo function like this:
`await navigateTo({
  path: '/categories/all',
  quuery: {
    region: 'US',
  }
});`

It wont work this way unless I provide different path like: /categories/popular?region=US. I want navigation to work just with query parameters on same path.

**Comment by danielroe** [comment]
Would you open an issue with a reproduction? I can't reproduce it.

**Comment by bot08** [comment]
Do U want refresh page after changing query parameter? If so, try to track the query and do action.
